about.html

```diff
@@ -0,0 +1,124 @@
+              <p class=" pass">The Fashion Show bring a joyful celebration believing in the value of openness and sharing, creating a postive change with people from over 80 countries is taking place Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+            </div>
+            <p class="fass">Please contact us per Email for any further questions about UC's Fashion Show, 2023</p>
+            <p class="las">uchennaoluchi49@gmail.com</p>
+        </div>
+    </header>
+    <section class="logo">
+        <h2 class="title">Uc's Commons <br>Fashion Show  2023 Logo </h2>
+    <div class="line"></div>
+    <div class="description">
+      <p class="des">The Logo of UC's Fashion Show 2023 was decided throught the logo competition from 8. June to 7. july.</p>
+      <p class="description1">Lorem ipsum dolor sit amet consectetur adipisicing elit. Esse dolorum tempore, qui quo temporibus aliquid ullam nihil recusandae natus, eius porro repellat aliquam sequi. Explicabo necessitatibus corporis molestias praesentium cumque! Lorem ipsum dolor sit amet consectetur adipisicing elit. Odit accusamus perferendis aperiam repellat porro facere.</p>
+    </div>
+    <img class="image" src="img/pngtree.jpg" alt="Film Festival logo">
+    </section>
+    <section class="past">
+        <h2>See the past US's Fashion Show</h2>
+        <div class="line"></div>
+        <p class="past-p">Take a look at the last two UC's Fashion Show which took place in USA and China</p>
+        <div class="past-img">
+          <div class="onn ooo">
+            <h2>2020</h2>
+            <p>UC's Fashion Show 2020 in USA</p>
+          </div>
+          <div class="onn ttt">
+            <h2>2018</h2>
+            <p>UC's Fashion Show 2018 in China</p>
+          </div>
+        </div>
+    </section>
+    <section class="sponsors">
+        <h2>Sponsors</h2>
+        <div class="line"></div>
+    
+        <div class="contain">
+    
+          <div class="imgg">
+            <img src="img/aptLearn-horizontal-locukup-colored.png" alt="" srcset="">
+          </div>
+    
+          <div class="imgg">
+            <img src="img/google.png" alt="" srcset="">
+          </div>
+    
+          <div class="imgg">
+            <img src="img/Google-flutter-logo.png" alt="" srcset="" style="background-color: white;">
+          </div>
+    
+          <div class="imgg">
+            <img src="img/coke.png" alt="" srcset="" >
+          </div>
+    
+          <div class="imgg">
+            <img src="img/paystack-logo-vector.png" alt="" srcset="">
+          </div>
+        </div>
+    </section>
+    
+    <footer class="footer">
+      <img class="footer-image" src="img/woman-fashion-logo-design-vector-44166694.jpg" alt="Logo of UC's Fashion Show">
+      <div class="info">
+        <h4 class="footer-title">Uc's Fashion Show 2023</h4>
```

Comment:
- [ ] Kindly fix the space between the image and the paragraph in the footer
<img width="1440" alt="Screenshot 2023-01-04 at 20 40 32" src="https://user-images.githubusercontent.com/81444338/210703122-e6143100-0379-42d6-ab26-1faffaa5da14.png">